docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -110,6 +110,20 @@ is used.
   * - ``iceberg.max-partitions-per-writer``
     - Maximum number of partitions handled per writer.
     - 100
+
+ORC format configuration
```

Comment:
I would drop the first commit because the `orc` prefix in the property name makes it fairly obvious that the config is ORC specific.

Reply:
To maintain backward compatibility, the same parameters are used. For details, please refer to the following link.
https://github.com/trinodb/trino/pull/11732#discussion_r869980722

Reply:
Thanks, I will not rename `hive.orc.bloom-filters.enabled`.